velox/dwio/parquet/tests/ParquetWriterTestBase.h

```diff
@@ -24,6 +24,7 @@
 #include "velox/dwio/common/tests/utils/DataFiles.h"
 #include "velox/dwio/parquet/writer/Writer.h"
 #include "velox/exec/tests/utils/TempDirectoryPath.h"
+#include "velox/vector/fuzzer/VectorFuzzer.h"
```

Comment:
Can we now remove 'include' for BatchMaker?

Reply:
Removed

Reply:
@duanmeng For some reason, I still see

#include "velox/dwio/common/tests/utils/BatchMaker.h"